cmd/soroban-cli/src/commands/contract/deploy/wasm.rs

```diff
@@ -218,25 +228,43 @@ impl NetworkRunnable for Cmd {
             salt,
             &key,
         )?;
+
         if self.fee.build_only {
+            output.check("Transaction built!");
             return Ok(TxnResult::Txn(txn));
         }
 
+        output.info("Simulating deploy transaction…");
+
         let txn = client.simulate_and_assemble_transaction(&txn).await?;
         let txn = self.fee.apply_to_assembled_txn(txn).transaction().clone();
+
         if self.fee.sim_only {
+            output.check("Done!");
             return Ok(TxnResult::Txn(txn));
         }
+
+        output.globe("Submitting deploy transaction…");
+        utils::log_transaction(&output, &txn, &network, true)?;
```

Comment:
Given that this function requires `output` does it make sense to make it an instance method?

Reply:
thought about it, but didn't have a strong opinion. I'll move it over to the `Output` struct then.